app/models/user.rb

```diff
@@ -0,0 +1,53 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  comments_count         :integer          default(0)
+#  email                  :citext           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  likes_count            :integer          default(0)
+#  name                   :string
+#  photos_count           :integer          default(0)
+#  private                :boolean          default(TRUE)
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  username               :citext
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_username              (username) UNIQUE
+#
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :own_photos, class_name: "Photo", foreign_key: "owner_id"
+
+  has_many :comments, foreign_key: "author_id"
+
+  has_many :sent_follow_requests, foreign_key: :sender_id, class_name: "FollowRequest"
```

Comment:
Can you add an option to prevent getting an error for the foreign key constraint if a user gets deleted and leaves associated records with a user no longer existing?